js/models/messagecollection.js

```diff
@@ -21,6 +21,20 @@ define(function(require) {
 		model: Message,
 		comparator: function(message) {
 			return message.get('dateInt') * -1;
+		},
+		initialize: function() {
+			this.on('add', function(message) {
```

Comment:
in regards to consistent coding style and testability I'd suggest to move this anonymous callback to a method of this class and just have the wiring in the constructor ;)